packages/datadog-plugin-apollo-gateway/test/naming.js

```diff
@@ -0,0 +1,19 @@
+const { resolveNaming } = require('../../dd-trace/test/plugins/helpers')
+
+const rawExpectedSchema = {
+  server: {
+    v0: {
+      opName: 'apollo-gateway.request',
```

Comment:
I think it would be more in-line with other plugins to use `apollo.gateway.request`, although I'm not 100% sure.

@jbertran Thoughts on this? Basically Apollo has 3 types of components exposed, `server`, `client` and `gateway` (which is basically a proxy).

Reply:
Good spot, thanks! We already have this naming convention for `moleculer` (and less relevant, but cloud providers too), so `apollo.{gateway|server|client}.request` makes complete sense.